content-services/community/install/index.md

```diff
@@ -32,9 +32,9 @@ See [Install with zip]({% link content-services/community/install/zip/index.md %
 
 Ansible playbooks provide an automated way of deploying a complete Community Edition system on "bare metal" servers, Virtual Machines, or Virtual Private Cloud instances, without the use of containers. The single playbook provided can be configured to deploy single or multi-machine instances, with a number of supported configuration options.
 
-See [Install using Ansible](#LINK-content-services/community/install/ansible.md) for basic instructions on deploying with Ansible.
+See [Install using Ansible]({% link content-services/community/install/ansible.md %}) for basic instructions on deploying with Ansible.
```

Comment:
When you click this you get to the page, but there is no left navigation. Ansible is probably not in the toc.

Reply:
Ah - good spot - fix is in the next commit